fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm

```diff
@@ -49,6 +49,8 @@
 	return TRUE
 
 /datum/action/cooldown/bloodsucker/feed/DeactivatePower()
+	if(!active)
+		return
```

Comment:
This should never be the case, this is hiding an issue. Remove please.

Reply:
done